### PR TITLE
refactor(testing) Split out testing layout into different fragments. …

### PR DESCRIPTION
**Pull request by acconway**
…Changed layout. Added buttons for all inapp types. Refactored inapp ids.

### CODE REVIEW

**Comment by acconway** [comment]
Companion to https://github.com/Leanplum/Testing/pull/15/files

**Comment by acconway** [comment]
I made the buttons smaller and added sub menus for testing. I think a preferable version of this is probably to keep just one menu item for testing, have that open a view with buttons which lead to more views.

File: app/src/main/res/layout/testing_inapp.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (acconway) [inline app/src/main/res/layout/testing_inapp.xml:1]: @benmarten or @Enuviel Can you share your settings for xml formatting for android studio? I just formatted this with the default

**Comment by acconway** [comment]
I tested and appium doesn't seem to have a great way to deal with a scroll view. if it's not visible on screen it can't see buttons, and scrolling manually to find each button sounds painful. 

However, this is still annoying to have to change the layout as we add more buttons. This layout should have room for a while...

File: app/src/main/java/com/leanplum/android/example/fragments/TestingInAppFragment.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+    final Button interstitial = root.findViewById(R.id.interstitial);
+    interstitial.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        Leanplum.track("interstitial");
+      }
+    });
+
+    final Button push_pre_permission = root.findViewById(R.id.push_pre_permission);
```
Comment (Enuviel) [inline app/src/main/java/com/leanplum/android/example/fragments/TestingInAppFragment.java]: We do not have push pre-permission on Android - please remove this button

**Review by Enuviel — APPROVED** [review]

File: app/src/main/res/layout/testing_inapp.xml (hunk 1 above)
Comment (benmarten) [reply app/src/main/res/layout/testing_inapp.xml:1]: I think default is totally fine ;)

**Review by benmarten — APPROVED** [review]

**Pull request merged**